legend-engine-xts-graphQL/legend-engine-xt-graphQL-pure/src/main/resources/core_external_query_graphql/binding/fromPure/introspection/fromPure_Introspection.pure

```diff
@@ -66,6 +67,20 @@ function <<access.private>> meta::external::query::graphQL::binding::fromPure::i
                               )
       ])
     )
+  )->concatenate(
+    $types->map(t|
+      if($t->instanceOf(Class) && !$t->cast(@Class<Any>)->cast(@AnnotatedElement).stereotypes.value->contains('Query'),
+        | pair(
+            $t.name->toOne() + '_bool_exp',
+            ^__Type(
+              kind = __TypeKind.INPUT_OBJECT,
```

Comment:
In GraphQL, INPUT_OBJECT is used for objects in arguments. It's different from OBJECT, which is a query-able type.

Reply:
Excerpt from the spec, 

The GraphQL Object type ([ObjectTypeDefinition](https://spec.graphql.org/October2021/#ObjectTypeDefinition)) defined above is inappropriate for re-use here, because Object types can contain fields that define arguments or contain references to interfaces and unions, neither of which is appropriate for use as an input argument. For this reason, input objects have a separate type in the system.